### PR TITLE
FBXLoader: Account for invalid parseAnimationLayers children IDs

### PR DESCRIPTION
**Pull request by jake**
For some of my FBX files, a few of the relationships point to objects without any relationship data. As a result, parseAnimationLayers crashes when accessing `fbxTree.Objects.Model[ modelID.toString() ]` and the FBX fails to parse.

Here are a few sample FBX that fail to parse as a result of the invalid data. The files were generated with [Unity's FBX Exporter plugin](https://docs.unity3d.com/Packages/com.unity.formats.fbx@2.0/manual/exporting.html), but are valid in every other FBX viewer or importer I have.

https://hypermesh-staging.accelerator.net/dev/performances/eeh5Y6?download=BNN-E021-A007.fbx (63MB)

Screen Shot 2020-05-15 at 4 35 26 PM

https://hypermesh-staging.accelerator.net/dev/performances/XdhjOd?download=BNN-E021-A010.fbx (28MB)

Screen Shot 2020-05-15 at 4 37 06 PM

Let me know if I can be of any help testing, or providing more detail. This goes more low-level than I'm familiar with, so I'm not sure if the bug is better fixed on a helper level. I've also left debugging log in there, as I figured it helps highlight the problem.

### CODE REVIEW

File: examples/js/loaders/FBXLoader.js [hunk 1]
```diff
@@ -2491,6 +2491,11 @@ THREE.FBXLoader = ( function () {
 									if ( modelID !== undefined ) {
 
 										var rawModel = fbxTree.Objects.Model[ modelID.toString() ];
+										
+										if (!rawModel) {
```
Comment (mrdoob) [inline examples/js/loaders/FBXLoader.js:2495]: What exactly are you testing for? `undefined`, `null`, `0`, `''`, ... ?

Comment (mrdoob) [reply examples/js/loaders/FBXLoader.js:2495]: This makes the code easier to read:

```js
if ( rawModel === undefined ) {
```

Comment (mrdoob) [inline examples/js/loaders/FBXLoader.js:2495]: This makes the code more readable:

```js
if ( rawModel === undefined ) {
```

Comment (jake) [reply examples/js/loaders/FBXLoader.js]: updated!

**Comment by mrdoob** [comment]
Thanks!

**Comment by zlf2002171** [comment]
unsubscribe

2020-05-18 

zlf2002171 



发件人：Jacob Bijani <notifications@github.com>
发送时间：2020-05-16 05:29
主题：Re: [mrdoob/three.js] Account for invalid parseAnimationLayers children IDs in FBXLoader (#19368)
收件人："mrdoob/three.js"<three.js@noreply.github.com>
抄送："Subscribed"<subscribed@noreply.github.com>

@jake commented on this pull request.



In examples/js/loaders/FBXLoader.js:
> @@ -2491,6 +2491,11 @@ THREE.FBXLoader = ( function () {
 									if ( modelID !== undefined ) {
 
 										var rawModel = fbxTree.Objects.Model[ modelID.toString() ];
+										
+										if (!rawModel) {

updated!

**Pull request merged**